eslint/Tests/EslintUnitTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\User\Block\Role\Tab;
```

Comment:
Please move the unit tests to   correct the `Magento2/Tests/Eslint` folder and correct the namespace